htdocs/core/modules/contract/doc/pdf_strato.modules.php

```diff
@@ -740,7 +740,7 @@ protected function _pagehead(&$pdf, $object, $showaddress, $outputlangs, $output
 			$this->recipient = $object->thirdparty;
 
 			// Recipient name
-			if ($usecontact && ($object->contact->socid != $object->thirdparty->id && (!isset($conf->global->MAIN_USE_COMPANY_NAME_OF_CONTACT) || getDolGlobalString('MAIN_USE_COMPANY_NAME_OF_CONTACT')))) {
+			if ($usecontact && $object->contact->socid != $object->thirdparty->id && getDolGlobalInt('MAIN_USE_COMPANY_NAME_OF_CONTACT')) {
```

Comment:
The 2 conditions are not the same.
If option never set, we want to act as if szt to 1 (default value)